### PR TITLE
Fix #2482 make tarball

### LINKED ISSUE #2482: make-tarball script no longer works
/admin/make-tarball has succumbed to bit-rot due to no longer being used.

We now have a requirement for rolling our own tarballs in-house.

The script is failing like this:

```
$ ./admin/make-tarball 
./admin/make-tarball: line 38: cylc: command not found
ERROR
```

### PR DESCRIPTION
**Pull request by jontyq**
Fixes #2482

### CODE REVIEW

File: admin/make-tarball [hunk 1]
```diff
@@ -35,11 +35,9 @@ if [[ ! -d .git && ! -f .git ]]; then
     exit 1
 fi
 
-VERSION=$( cylc -v )
-if [[ $VERSION = *-dirty ]]; then
-    echo "The work tree is not clean: ABORTING" >&2
-    exit 1
-fi
+THIS_DIR=$( cd $( dirname $0 ); pwd; )
```
Comment (hjoliver) [inline admin/make-tarball]: Just curious, compared to plain old `THIS_DIR=$(dirname $0)` this construct makes the path absolute if the script is invoked by a relative path - but I don't think that matters in this context.  Is there some other advantage?

Comment (hjoliver) [reply admin/make-tarball]: I agree that that removing the "-dirty" abort is good, now that this script is no longer used to make official releases, and "-dirty" appears in the tarball name, but you just need to update (delete) the last paragraph of the command usage string accordingly.

Comment (jontyq) [reply admin/make-tarball]: Re the path thing, it is typically what I do to prevent bad corner cases.

Comment (jontyq) [reply admin/make-tarball]: Have updated the usage string.

**Comment by hjoliver** [comment]
No need for a second review on this one, as a small change that does not affect users.

**Pull request merged**